psi_cardinality/javascript/bindings/server.cpp

```diff
@@ -43,7 +56,9 @@ EMSCRIPTEN_BINDINGS(PSI_Server) {
                 }))
       .function("GetPrivateKeyBytes",
                 optional_override([](const PSICardinalityServer &self) {
-                  std::string bytes = self.GetPrivateKeyBytes();
-                  return bytes;
+                  const std::string bytes = self.GetPrivateKeyBytes();
+                  const std::vector<std::uint8_t> key_vector(bytes.begin(), bytes.end());
+                  emscripten::val key_bytes  = emscripten::val::array(key_vector.begin(), key_vector.end());
```

Comment:
Again, shouldn't reading from the string directly work?
```
emscripten::val key_bytes  = emscripten::val::array(bytes.begin(), bytes.end());
```

Reply:
I’ll test. Wanted to ensure it was going to return a Uint8 type for each element.